src/components/OntologyBrowser/SelectionTree.tsx

```diff
@@ -55,20 +55,20 @@ export class SelectionTree extends Component<SelectionTreeProps, SelectionTreeSt
     treeNodes: TreeNode[] = [],
     checkedKeys: string[] = [],
     targetKeys: string[] = [],
-    disabled: boolean = false,
+    treeDisabled: boolean = false,
   ): TreeNode[] =>
     treeNodes
-      .map(({ children, key, title, results, hidden, exactTagCount }: TreeNode) => {
+      .map(({ children, key, title, results, hidden, exactTagCount, disabled }: TreeNode) => {
```

Comment:
would it be a bit cleaner to process the chain map + filter by a reduce instead (it'd remove a loop)